### PR TITLE
Clarify role of pricing council

### PR DESCRIPTION
**Pull request by abschu**

### CODE REVIEW

File: content/departments/product-engineering/product/process/pricing.md
Comment (jasonyavorsky) [inline content/departments/product-engineering/product/process/pricing.md:82]: How do you make a request for a decider to the pricing council? Email the group? Create a ticket? Ask in Slack? Something else?

File: content/departments/product-engineering/product/process/pricing.md
Comment (jasonyavorsky) [inline content/departments/product-engineering/product/process/pricing.md:68]: How long before a decision is a recommendation due?

File: content/departments/product-engineering/product/process/pricing.md
Comment (jasonyavorsky) [inline content/departments/product-engineering/product/process/pricing.md:92]: What is the lead time for the pricing council to make a decision from the time you make a request? I.e., if I have a feature I want to go GA, how long before that date should the recommendation go to the pricing council? How long before the GA date should you make an official request for a decider?

File: content/departments/product-engineering/product/process/pricing.md
Comment (jasonyavorsky) [inline content/departments/product-engineering/product/process/pricing.md:87]: One question - revenue is a top focus for the company, and so in order to decide to work on something you sort of already have to know well in advance of even starting development how it will be priced and what the expected ROI is. What is the role of the pricing council in answering those questions? At the moment it seems we mostly wait until just before GA to make a pricing decision, but depending on that decision we may not have ever even started the work (if you see what I mean). Thoughts?

File: content/departments/product-engineering/product/process/pricing.md
Comment (Joelkw) [reply content/departments/product-engineering/product/process/pricing.md:82]: +1! The way we did this our time around was meet 1:1 with people in every category here (Gregg, Christina, Steph, Andy) and ultimately land at which person (Gregg in our case) needed to approve the pricing. 

Perhaps a quick process for not only where to ask, but also what you should have ready when you make the ask would be helpful? From my experience, it was helpful to have the conversations about who should approve the final decision after doing a lot of the initial work / initial RFC, so council members could get a sense of which areas were the biggest question areas (for example: are the main questions about product and where to draw the free / not free line? Are the main questions about the dollar price? Are the main questions about packaging?)

**Review by Joelkw — COMMENTED** [review]
Added a quick thought, love that we're defining this more!

**Review by stephmhay — APPROVED** [review]

File: content/departments/product-engineering/product/process/pricing.md
Comment (jasonyavorsky) [inline content/departments/product-engineering/product/process/pricing.md:66]: I think there is also a gap around "what is the current strategy of the pricing council". That can be an internal doc for sure, but we should have something that people who are about to work with the pricing council can read in order to have some framework for making their proposal. It would include things like what are tiers are and why, what goes in what tiers, maybe how tiers are enforced, that sort of thing. Otherwise the recommendations people will produce will be total guesses.

**Comment by jasonyavorsky** [comment]
Hey @abschu is this something that's still being worked on or are we going in another direction with the pricing council/making pricing decisions?

**Comment by jasonyavorsky** [comment]
@abschu do you know if there's anyone who can help put the finishing touches on this one?

**Comment by jasonyavorsky** [comment]
Given this has stalled out the last few months I'm going to stop tracking this effort.

**Pull request closed without merging**
